webdriver-spec.html

```diff
@@ -1435,288 +1435,532 @@
+  implementation <em>MAY</em> accept a hostname and port through this property.
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>httpProxyPort</code>"
+<td>number
+<td>Defines the proxy <em>port</em> for HTTP traffic.
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>sslProxy</code>"
+<td>string
+<td>Defines the proxy <em>hostname</em> for encrypted SSL traffic.
+  Alternatively, an implementation <em>MAY</em> accept a hostname and port
+  through this property.
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>sslProxyPort</code>"
+<td>number
+<td>Defines the proxy <em>port</em> for encrypted SSL traffic.
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>socksProxy</code>"
+<td>string
+<td>Defines the proxy <em>hostname</em> for a SOCKS proxy. [[RFC1928]]
+  Alternatively, an implementation <em>MAY</em> accept a hostname and port
+  through this property.
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>socksProxyPort</code>"
+<td>number
+<td>Defines the proxy <em>port</em> for a SOCKS proxy. [[RFC1928]]
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>socksVersion</code>"
+<td>number
+<td>Defines the SOCKS proxy version.
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>socksUsername</code>"
+<td>string
+<td>
+  Defines the username used when authenticating with a SOCKS proxy. [[RFC1928]]
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>socksPassword</code>"
+<td>string
+<td>
+  Defines the password used when authenticating with a SOCKS proxy. [[RFC1928]]
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+  This property <em>MUST NOT</em> be set by a <a>remote end</a> when returning
+  <a>capabilities</a> to the <a>local end</a>.
+</tr>
+
+</tbody>
+</table>
+</section><!-- /Proxy Configurations -->
 
-  <dl>
-   <dt>"<code>browserName</code>"
-   <dd>Lowercase name of the user agent
+<section>
+<h3>Session Timeouts Configuration</h3>
+<p>The <a>Session Timeouts Configuration</a> capability is a JSON Object nested
+within the primary capabilities. It consists of the properties listed in the
+table below. These values may be changed after creating a <a>new session</a>
+using the <a>Set Timeout</a> command.
```

Comment:
These timeouts are all in the `Set Timeout` section. Refer to these rather than duplicate the information?